apps/rollup/bundle.mjs

```diff
@@ -109,6 +109,7 @@ function createConfig(packageDir, format) {
         dir: `${packageDir}/dist/${format}`,
         format: format,
         entryFileNames: `[name].${{ esm: 'mjs', cjs: 'cjs' }[format]}`,
+        sourcemap: tsconfig.config.compilerOptions.sourceMap,
```

Comment:
Why we need this change?

Reply:
Because of this: https://github.com/rollup/plugins/blob/typescript-v12.1.2/packages/typescript/src/options/validate.ts#L22

as mentioned above:

> * This removes the following warning: ![image](https://github.com/user-attachments/assets/6f2bfeaf-9033-4f07-a922-3dfeba348abc)